src/css/main.less

```diff
@@ -1478,3 +1472,7 @@ h3.small-caps {
     padding: 0;
   }
 }
+
+.task-details dt {
+  text-align: left;
```

Comment:
True, the about modal doesn't look good with left-align.
But now we diverge somehow with listings in the layout.